docs/logging-and-managing-experiment-results/index.rst

```diff
@@ -2,101 +2,153 @@
-   Downloading experiment data programmatically <downloading-experiment-data-programmatically/index.rst>
+Generic recipe being:
 
-.. External links
+.. code-block:: python3
 
-.. |Youtube Video| raw:: html
+    neptune.log_X('some_name', some_value)
 
-    <iframe width="720" height="420" src="https://www.youtube.com/embed/w9S5srkfSI4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+Where ``X`` could be metric, artifact, chart, pickle, etc. Check the :ref:`logging section <what-you-can-log>` for a complete list.
 
-.. |neptune-client-github| raw:: html
+.. note::
 
-    <a href="https://github.com/neptune-ai/neptune-client" target="_blank">Neptune client</a>
+    If you work in Notebooks, you need to place :meth:`~neptune.experiments.Experiment.stop` at the very end of your experiment to make sure that everything will be closed properly.
+
+    Note that you are still able to :ref:`update an experiment <update-existing-experiment>` that was closed before.
+
+Essential Neptune client concepts
+---------------------------------
+:ref:`Project <logging_project>` and :ref:`Experiment <logging_experiment>` are two important entities in Neptune.
+
+Basic snippet below, sets project and creates new experiment in that project.
+
+.. code-block:: python3
+
+    # Set project
+    neptune.init('my_workspace/my_project')
+
+    # Create new experiment
+    neptune.create_experiment()
+
+.. _logging_project:
+
+Project
+^^^^^^^
+It is a **collection of Experiments**, created by user (or users) assigned to the project.
 
-.. |Neptune| raw:: html
+You can log experiments to the project or fetch all experiments that satisfy some criteria.
 
-    <a href="https://neptune.ai" target="_blank">Neptune</a>
+.. code-block:: python3
 
-.. |onboarding| raw:: html
+    # Set project and get project object
+    project = neptune.init('my_workspace/my_project')
 
-    <a href="https://ui.neptune.ai/shared/onboarding/experiments" target="_blank">shared/onboarding</a>
+    # Use project to create experiment
+    project.create_experiment()
 
-.. |github-issues| raw:: html
+    # Use project to get experiments data from the project
+    project.get_leaderboard(state=['succeeded'])
 
-    <a href="https://github.com/neptune-ai/neptune-client/issues" target="_blank">GitHub issues</a>
+Learn more about :ref:`downloading data from Neptune <guides-download_data>`. Check also, :class:`~neptune.projects.Project` to use in your Python code.
 
-.. |Example1| raw:: html
+.. _logging_experiment:
 
-    <a href="https://ui.neptune.ai/USERNAME/example-project/e/HELLO-48/source-code?path=.&file=classification-example.py" target="_blank">Example</a>
+Experiment
+^^^^^^^^^^
+Experiment is everything that you log to Neptune, beginning at ``neptune.create_experiment()`` and ending when script finishes or when you explicitly stop the experiment with :meth:`~neptune.experiments.Experiment.stop`.
```

Comment:
again `stop` is displayed and it should be `experiment.stop` or `neptune.stop` I think

Reply:
right.

Reply:
change it a bit.